src/components/utility/UtilitySrv.js

```diff
@@ -0,0 +1,21 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.utility').factory('UtilitySrv', [
+        function () {
+            var UtilitySrv = {};
+
+            UtilitySrv.general = {};
+
+            UtilitySrv.general.createGuid = function(){
```

Comment:
it will be nice to have test for it.....generate 10 quid and ensure not identical..not mandatory